README.md

```diff
@@ -52,31 +51,29 @@ async fn main() {
 
 ```
 
-For a longer example, check out [Vertex](https://github.com/Restioson/vertex/tree/development), a chat application
-written with xtra and spaad on the server.
-
-Too verbose? Check out the [spaad](https://crates.io/crates/spaad) sister-crate!
+For a longer example, check out [Vertex](https://github.com/Restioson/vertex/tree/development), a chat application written with xtra and spaad on the server.
 
 ## Okay, sounds great! How do I use it?
-Check out the [docs](https://docs.rs/xtra) and the [examples](https://github.com/Restioson/xtra/blob/master/examples)
-to get started! Enabling the `tokio`, `async_std`, `smol`, or `wasm_bindgen` features
-is recommended in order to enable some  convenience methods (such as `xtra::spawn_tokio`). Which you enable will depend on
-which executor you want to use (check out their docs to learn more about each). If you have any questions, feel free to
-[open an issue](https://github.com/Restioson/xtra/issues/new) or message me on the [Rust discord](https://bit.ly/rust-community).
+
+Check out the [docs](https://docs.rs/xtra) and the [examples](https://github.com/Restioson/xtra/blob/master/examples) to get started!
+Enabling the `tokio`, `async_std`, `smol`, or `wasm_bindgen` features is recommended in order to enable some  convenience methods (such as `xtra::spawn_tokio`).
+Which you enable will depend on which executor you want to use (check out their docs to learn more about each).
+If you have any questions, feel free to [open an issue](https://github.com/Restioson/xtra/issues/new) or message me on the [Rust discord](https://bit.ly/rust-community).
```

Comment:
I changed this to one sentence per line to make working with github's suggestion easier.